zerver/tests/test_docs.py

```diff
@@ -74,7 +74,6 @@ def test_doc_endpoints(self) -> None:
         self._test('/errors/404/', 'Page not found')
         self._test('/errors/5xx/', 'Internal server error')
         self._test('/emails/', 'manually generate most of the emails by clicking')
-        self._test('/register/', 'Sign up for Zulip')
```

Comment:
I think you might just need to add `landing_page=False` on this?

Reply:
The `_test` with subdomain as "zulip" results in 200 while with subdomain "" results in 302 as it there is no realm there.